src/App.js

```diff
@@ -1,8 +1,23 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
 
 const App = () => (
-  <h1>React sum</h1>
+  <React.Fragment>
```

Comment:
можно использовать короткую запись
```suggestion
  <>
```